### ISSUE TITLE
npm ERR! missing script: emulate

### ISSUE THREAD

**Issue by qalqi** [prescottprue/generator-firebase-functions#3]
```
yo firebase-functions:function uppercaser
npm run functions:emulate```

> npm run emulate
> npm ERR! missing script: emulate

**Comment by prescottprue** [comment]
Try `npm run functions:start` - `npm run functions:emulate` is old and no longer part of the template. Are you sure you have updated recently? `npm i -g generator-react-firebase`

**Comment by qalqi** [comment]
https://github.com/prescottprue/generator-firebase-functions/blob/8205b02d5eb14e6b6e41baefcbd86aed00bdda82/generators/app/templates/package.json#L10

**Comment by prescottprue** [comment]
@qalqi That is no longer part of the template which is generated, and the link you provided actually points to an old commit. Are you generating a new project or is this for an existing project? Did you try updating?

**Comment by qalqi** [comment]
I was trying to use `generator-firebase-functions` instead of  `generator-react-firebase`

Wont just `generator-firebase-functions` work standalone?

**Comment by prescottprue** [comment]
Ahh I understand - I thought this was actually on the main generator repo. Thanks for clarifying

**Comment by qalqi** [comment]
https://github.com/prescottprue/generator-firebase-functions/blob/8205b02d5eb14e6b6e41baefcbd86aed00bdda82/generators/function/templates/_httpsFunction.js#L16

Also, This should be 
```
response.write('Hello from <%= name %>')
response.end()
```

**Comment by prescottprue** [comment]
@qalqi Since firebase http cloud functions are express wrapped functions, `response.send()` should also work right? Have you experienced it not working?

Either way, thanks for the suggestions

**Comment by qalqi** [comment]
Yep. got error  `500, Error: Can't set headers after they are sent`.. after executing writeHead, we cant response.send it seems.  instead response.write or response.end

https://stackoverflow.com/a/7086621/2560946

**Comment by prescottprue** [comment]
Agreed - `response.status().send()` is the right way to do it, so I'll just switch to `response.end`

**Comment by qalqi** [comment]
Waiting for new release :)
